src/routes/books.ts

```diff
@@ -1,15 +1,16 @@
 import { NextFunction, Request, Response, Router } from "express";
 import * as _ from 'lodash';
 
-import { BookFactory } from '../model/book-factory';
+import { BookFactory, PLACEHOLER_IMG } from '../model/book-factory';
 import { BooksStore } from '../books-store';
 import { HTTP } from './http';
+import { NotificationService } from "../notification-service";
 
 export class BooksRoute {
 
-  public static create(router: Router, bookStore: BooksStore) {
+  public static create(router: Router, bookStore: BooksStore, notificationService: NotificationService) {
 
-    let booksRoute = new BooksRoute(bookStore);
+    let booksRoute = new BooksRoute(bookStore, notificationService);
```

Comment:
`const` everywhere! ✌️ Or is there a reason for why we used `let` so often? @JohannesHoppe

Reply:
Kein Grund. Weil's normalerweise egal ist! 😉